src/Contracts/User.php

```diff
@@ -38,4 +38,11 @@ public function getEmail();
      * @return string
      */
     public function getAvatar();
+
+    /**
+     * Get the gender of the user.
+     *
+     * @return string
+     */
+    public function getGender();
```

Comment:
> Absolutely not.

It needs to, because adding a new method to an interface is a major breaking change.